client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfiguration.java

```diff
@@ -158,6 +159,6 @@ static Builder builder() {
          * but the backend is not, it makes no sense to redirect the caller to a new backend. The client will still
          * follow redirects.
          */
-        DANGEROUS_DISABLE_AUTOMATIC_RETRY_ON_QOS;
```

Comment:
It's not actually dangerous in the same way that `DANGEROUS_DISABLE_SYMPATHETIC_CLIENT_QOS` is (that has the risk of overloading servers).